### ISSUE TITLE
Change wagtailadmin/css/core.css to allow flexible logo height in admin

### ISSUE THREAD

**Issue by mainderay** [wagtail/wagtail#3517]
# Feature Request

### Issue Summary

Using a custom logo in the admin page that is too tall results in the search box being pushed down into the main nav menu.

### Steps to Reproduce

Add a logo that is say 200px in height.

### Proposed Solution

Consider changing the below section in wagtailadmin/css/ core.css from:

```
 .nav-main {
    position: absolute;
    top: 175px;
    margin-bottom: 127px;
  }
```

to

```
 .nav-main {
    position: relative;
    top: 12px;
    margin-bottom: 127px;
  }
```

**Comment by thibaudcolas** [comment]
Hey @mainderay, sorry we're only getting to this now.

What you're proposing looks very sensible. I'm not sure why this has been positioned like this in the past, we may want to check multiple variations of viewport sizes to make sure the change caters to all scenarios.

I think it should also be documented somewhere what custom logo sizes will work, if that's not the case already. I don't know this feature very well.

Would you be keen to submit a pull request with those changes? Doesn't need to be perfect, but a first iteration would help a lot in getting this done.

**Comment by thibaudcolas** [comment]
If anyone is interested in getting this fixed, #3798 needs further work.

**Comment by thibaudcolas** [comment]
I believe this has been fixed via #4747.
